README.md

```diff
@@ -1,37 +1,91 @@
+If these don't meet your specific needs, please contact the PlatUI team via [#team-plat-ui](https://hmrcdigital.slack.com/messages/team-plat-ui/) to discuss.
+
+#### Enabling users to report technical issues with the service
+You should use the
+[HmrcReportTechnicalIssueHelper](https://github.com/hmrc/play-frontend-hmrc?tab=readme-ov-file#helping-users-report-technical-issues)
+to add a link to your service's main layout template, to enable users to report technical issues from any page in the service.
+
+For an example of the resulting form, please see the [Appendix](#contacting-hmrc---is-this-page-not-working-properly).
+
+#### Enabling users to provide feedback on a service in beta
+When your new service is in beta, you should use the
+[Beta Feedback banner](https://github.com/hmrc/play-frontend-hmrc?tab=readme-ov-file#adding-a-beta-feedback-banner)
+to add a banner that tells the user the service is in beta, and includes a link for them to provide feedback.
+
+For an example of the resulting form, please see the [Appendix](#providing-beta-feedback-about-services).
+
+### Other services
+Services that don't have a frontend (eg. mobile or API services) should instead integrate directly with the
+[deskpro-ticket-queue](https://github.com/hmrc/deskpro-ticket-queue) API.
+
+### User details attached to submitted tickets
+In addition to the information provided by the user, the service collects the following context data:
+* **Referrer** - this is the URL of the page on which the user initiated the contact journey, as passed on the
+  URL as a query string parameter. Deskpro uses it to classify issues. In the absence of a `referrerURL` parameter,
+  the value will be taken from the HTTP `Referer` header.
+* **HTTP UserAgent header** - this tells us what browser the user is using.
+* **Javascript?** - whether the user's browser has Javascript enabled
+* **authId** - as provided in the HeaderCarrier object
+* **sessionId** - as provided in the HeaderCarrier object
+* **User tax identifiers** - if the user is signed-in, any related tax identifiers will be attached to the tickets.
+  These are stored in `deskpro-ticket-queue` and can later be viewed from Deskpro.  They can include, but are not limited to:
+    * National Insurance Number (NINO)
+    * Unique Taxpayer Reference (UTR)
+    * VAT registration number
+    * PAYE reference
+
+[[Back to the top]](#top)
+
+## Further information
+
+### Getting help
+* [#team-platui](https://hmrcdigital.slack.com/messages/team-plat-ui/) - PlatUI is responsible for contact-frontend
+* [#team-ddcops](https://hmrcdigital.slack.com/messages/team-ddcops) - DDCOps is responsible for Deskpro
+
+### Maintenance documentation
+Maintenance documentation for the owning team, including architectural decision records (ADRs) can be found [here](docs/maintainers.md).
+
+[[Back to the top]](#top)
+
+## Appendix - Example forms
+
+### Contacting HMRC - *Is this page not working properly?*
```

Comment:
Re-ordered to put the 2 service-team-facing ones first; re-ordered/rewrote a few bits of copy to make more concise